src/Manager.js

```diff
@@ -183,7 +183,7 @@ class GiveawaysManager extends EventEmitter {
             if (typeof options.prize !== 'string') {
                 return reject(`options.prize is not a string. (val=${options.prize})`);
             }
-            if (!Number.isInteger(options.winnerCount) || !options.winnerCount > 0) {
+            if (!Number.isInteger(options.winnerCount) && options.winnerCount <= 0) {
```

Comment:
```suggestion
            if (!Number.isInteger(options.winnerCount) || options.winnerCount < 1) {
```

Reply:
Oh yes Number.isInteger is present thanks!

Reply:
I think using an `&&` statement would be better here.

Reply:
Since if the winner count is zero it will not throw an error.

Reply:
bro we are checking if is is not a number OR if the number is below 1
your change would check "only if it is not a number AND below 1 then return"

Reply:
Ahhhhh Alrighty thanks for correcting me xD